base/doc/ltnews42.tex

```diff
@@ -357,6 +357,42 @@ \subsection{Support separate font families for script fonts}
 
 \section{Code improvements}
 
+\subsection{Ensure that commands without arguments are not \cs{long}}
+
+In its original implementation \cs{newcommand} or \cs{renewcommand}
+always defined commands using \cs{long}\cs{def} even if the commands
+had no arguments, i.e., in situations where the concept of \cs{long}
+made no sense whatsoever.
+
+The issue with that behavior is that
```

Comment:
US Spelling.  OK?

Reply:
basically we go for US spelling for a long time, so yes I would say.